common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -955,14 +956,14 @@ def get_course(self, course_id, depth=0, **kwargs):
             raise ItemNotFoundError(course_id)
         return self._get_structure(course_id, depth, **kwargs)
 
-    def get_library(self, library_id, depth=0, **kwargs):
+    def get_library(self, library_id, depth=0, head_validation=True, **kwargs):
```

Comment:
You don't necessarily need to add `head_validation` as an explicit argument here, but I guess it makes the possibilities more clear to people reading the code.